webapp/src/pages/ReponseDetails/components/ReviewModal.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
```

Comment:
This component doesn't seem to be used? Also seems to be a copy of ResponsePage/components/ReviewModal.js ?